Assets/Scripts/CartesianTiles/WFSGeoJSONTileDataLayer.cs

```diff
@@ -0,0 +1,144 @@
+
+using System.Collections;
+using UnityEngine;
+using UnityEngine.Networking;
+using Newtonsoft.Json;
+using GeoJSON.Net.Feature;
+using Netherlands3D.Twin.Layers;
+
+namespace Netherlands3D.CartesianTiles
+{
+    /// <summary>
+    /// A custom CartesianTile layer that uses the cartesian tiling system to 'stream' parts of 
+    /// a WFS service to the client using the 'bbox' parameter.
+    /// The Twin GeoJSONLayer is used to render the GeoJSON data.
+    /// </summary>
+	public class WFSGeoJSONTileDataLayer : Layer
```

Comment:
Do I understand correctly that this code is only used as a piggyback to determine tiles bbox' and pass that onto the layer? Is it possible to make this a reusable thing, decouple it from WFS/GeoJSON so that we can reuse this same mechanism in other places, such as WMS?

Reply:
See comment: https://github.com/Netherlands3D/twin/pull/198#discussion_r1690849763